File: app/src/main/java/com/home/young/myPassword/application/BaseFragment.java
package com.home.young.myPassword.application;

import android.app.Fragment;
public class BaseFragment extends Fragment {

    protected BaseActivity getBaseActivity() {
        return (BaseActivity) getActivity();
    }

    protected void showToast(int resId) {
        getBaseActivity().showToast(resId);
    }

    protected void showToast(int resId, int duration) {
        getBaseActivity().showToast(resId, duration);
    }
}

File: app/src/main/kotlin/org/tianjyan/pwd/application/BaseFragment.kt
package org.tianjyan.pwd.application

import android.app.Fragment
import com.home.young.myPassword.application.BaseActivity

open class BaseFragment : Fragment() {

    protected val baseActivity: BaseActivity
        get() = activity as BaseActivity

    protected fun showToast(resId: Int) {
        baseActivity.showToast(resId)
    }

    protected fun showToast(resId: Int, duration: Int) {
        baseActivity.showToast(resId, duration)
    }
}